03LessonThree/homework/git-scm.html

```diff
@@ -31,12 +33,11 @@
                     <li>Disposable Experimentation. Create a branch to experiment in, realize it's not going to work, and just delete it - abandoning the work—with nobody else ever seeing it (even if you've pushed other branches in the meantime).</li>
                 </ul>
             </div>
-            <div id="article">
+          </article>
                 <h3>Community</h3>

```

Comment:
I believe this needs to be included inside an <article>, please re-work this.
It seems like you deleted tags accidentally. See for the source: view-source:https://robertowebdev.github.io/intro-workbook/03LessonThree/homework/git-scm.html